server/ops/docker/build-static-pikchr

```diff
@@ -0,0 +1,10 @@
+# build pikchr binary
+FROM alpine:3.12.3
+
+RUN apk update && apk add ca-certificates wget build-base
+
+WORKDIR build
+
+RUN wget https://pikchr.org/home/raw/447c7967a4063193bdbdeb85e4f26052aeff18f58bceef4eff2b0415f3ba39d0 -O pikchr.c
```

Comment:
Use https://pikchr.org/home/raw/7269f78c4a3aa2809bd8c278e522c4eac5568ad0fd6ed0a3f807f4a2f6367ef0 (that contains the ability to read diagram from `stdin`)